pages/item/_itemId/index.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-container fluid class="pa-0 itemContainer">
+    <v-img
+      :src="getItemBackdrop(item.Id)"
+      class="d-flex align-end backdropImage"
+      max-width="100%"
+    >
+      <div class="d-flex flex-wrap itemDetailsContainer">
+        <div class="itemDetailsLeft">
+          <v-img
+            v-if="item.ImageTags && item.ImageTags.Logo && getAspectRatio() > 1"
```

Comment:
This isn't responsive.

Like for the main artwork, we can use a Resize directive: https://vuetifyjs.com/en/directives/resizing/

Reply:
It should be, when using the poster, the item logo shouldn't be visible.